auth-web/src/components/auth/Statements.vue

```diff
@@ -35,22 +37,24 @@
           </div>
         </template>
         <template v-slot:[`item.action`]="{ item }">
-          <div class="btn-inline">
+          <div>
             <v-btn
               outlined
-              small
               color="primary"
               class="font-weight-bold mr-2"
+              aria-label="Download CSV"
+              title="Download statement as a CSV file"
```

Comment:
long button labels may affect the responsiveness in the smaller resolutions, can you make sure thats not affected here?

Reply:
This is an aria-label and title property - not the string inside the button